### ISSUE TITLE
[Query regarding the website's UI]

### ISSUE THREAD

**Issue by ctrlaltvikas** [AgarwalPragy/GATE2016_MarksEvaluator#3]
Hey @AgarwalPragy . I guess the emphasis behind creating this awesome application was to get more precision in marks vs rank for GATE_CSE. I would like to know if there is any UI/UX improvement required for making the webpage much better to which I can contribute to .
Please ping me back in this regard.

Thanks,

**Comment by AgarwalPragy** [comment]
Hey, that would be great help! My midsems just started, so let's discuss this after about a week?
Thanks :)

**Comment by ctrlaltvikas** [comment]
Yeah. That would be awesome. Looking forward to hearing soon man. :)

**Comment by ctrlaltvikas** [comment]
Have a look at this. I did some work on the index.php using material design lite.


![screenshot from 2017-03-01 02-33-31](https://cloud.githubusercontent.com/assets/12607959/23466937/98249e50-fec2-11e6-98a6-33475dd15f7b.png)

**Comment by anup-patel** [comment]
Working on This :)

![new_improved1](https://user-images.githubusercontent.com/21237713/33742434-54d8c4de-dbce-11e7-9425-43506daac8e4.PNG)
![2](https://user-images.githubusercontent.com/21237713/33742435-551adea0-dbce-11e7-9393-4f044757c593.PNG)

**Comment by AgarwalPragy** [comment]
Oh my God! That looks like a ton of work.
I'm excited, yet a little sad to inform that this project has been dropped in the favor of the Django based revamp we're planning this year.

This graph page will also be revamped, and we plan to simply use a Javascript based graphing library to get the job done.

Currently, we're working on the backend and the logic. I will seek your help once we reach the UI part.

Cheers :)
